### PR TITLE
denali search

### PR DESCRIPTION
**Pull request by ayeswarya**
[Resolves](url) #

## Description
Change Yavin reports search (dimensions and metrics) to denali search

## Proposed Changes
-

## Screenshots
Before
search-before
After
search-after
## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by kevinhinterlong — APPROVED** [review]

File: packages/reports/addon/templates/components/navi-list-selector.hbs [hunk 1]
```diff
@@ -3,17 +3,15 @@
   <h3 class="navi-list-selector__title" title="{{@title}}">{{@title}}</h3>
 </div>
 <div class="navi-list-selector__search">
-  <NaviIcon @icon="search" class="navi-list-selector__search-icon" />
-  <Input
+  <DenaliInput
+    @size="small"
+    @iconBack="search"
+    @iconBackClass="navi-list-selector__search-icon"
```
Comment (jkusa) [inline packages/reports/addon/templates/components/navi-list-selector.hbs:9]: Do we still need this class?

File: packages/reports/addon/templates/components/navi-list-selector.hbs [hunk 2]
```diff
@@ -3,17 +3,15 @@
...
+    value={{this.query}}
+    type="text"
     placeholder={{concat "Search " (capitalize @title)}}
```
Comment (jkusa) [inline packages/reports/addon/templates/components/navi-list-selector.hbs:12]:
```suggestion
    placeholder="Search {{capitalize @title}}"
```

File: packages/reports/app/styles/navi-reports/components/navi-list-selector.less [hunk 3]
```diff
@@ -71,14 +71,8 @@
...
-    &:focus {
-      border-bottom-color: @denali-brand-700;
-      border-bottom-style: solid;
-      border-bottom-width: 2px;
-    }
+    width: 230px;
+    max-width: 230px;
   }
 
   &__search-input-clear {
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/navi-list-selector.less:78]: Can we remove this now?

File: packages/reports/app/styles/navi-reports/components/navi-list-selector.less [hunk 4]
```diff
@@ -71,14 +71,8 @@
...
     transition: 200ms;
-    width: inherit;
-
-    &:active,
-    &:focus {
-      border-bottom-color: @denali-brand-700;
-      border-bottom-style: solid;
-      border-bottom-width: 2px;
-    }
+    width: 230px;
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/navi-list-selector.less:74]: Can most of the styles here now be removed? Also, can we clean up `&__search-icon`?

**Review by jkusa — APPROVED** [review]

**Pull request merged**